src/Sprites/Animation.h

```diff
@@ -20,8 +20,8 @@ struct Animation {
 		uint8_t optional3;
 		uint8_t optional4;
 
-		// Limited to size of uint16_t MAX
-		std::vector<Subframe> subframes;
+		// Maximum count of items in container is 128
```

Comment:
I suppose technically the range is 0..127 inclusive, so the "maximum count" is `127`.

Reply:
Wouldn't 0-127 be a count of 128 items? Maybe our terminology isn't agreeing here?

Reply:
Correct, there are 128 distinct bit combinations, but one of those counts is 0, hence the maximum is 127.